src/cpp/rtps/builtin/discovery/database/DiscoveryDataQueueInfo.hpp

```diff
@@ -18,7 +18,6 @@
  */
 
 #include <fastdds/rtps/common/CacheChange.h>
-#include <fastrtps/utils/fixed_size_string.hpp>
```

Comment:
The header is being removed without updating. And the fixed size string API is being used in this file.